aptos-move/aptos-vm/src/parallel_executor/vm_wrapper.rs

```diff
@@ -54,39 +98,33 @@ impl<'a, S: 'a + StateView> ExecutorTask for AptosVMWrapper<'a, S> {
         }
     }
 
-    fn execute_transaction(
+    fn execute_transaction_btree_view(
         &self,
-        view: &MVHashMapView<StateKey, WriteOp>,
+        view: &BTreeMap<StateKey, WriteOp>,
         txn: &PreprocessedTransaction,
+        txn_idx: usize,
```

Comment:
nit: probably can squeeze this in the view to be consistent

Reply:
as per our discussion: for a future refactoring, we can take some view related things from aptos vm and then we can include there and in parallel executor too, and we can unify these interfaces.